tests/py/runtests.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -eox pipefail
```

Comment:
Looks like the `x` might make tests fail? Is there another way for us to print the shell commands for future debugging?

Reply:
It doesn't cause the tests to fail--it's just unnecessarily verbose. What the user cares about is the output reports, not the command to generate the reports itself.

We don't use `-x` in other scripts, either. To debug the Makefiles and build files themselves, I think it's reasonable to make temporary changes to them.